README.md

```diff
@@ -12,43 +12,89 @@
-or correlate data from multiple sources using relational and time-series joins.
+It provides **high throughput ingestion** and **fast SQL queries**.
 
-We achieve high performance by adopting a column-oriented storage model, parallelized vector execution, SIMD instructions, and low-latency techniques.
-The entire codebase is built from the ground up in Java, C++ and [Rust](https://questdb.io/blog/leveraging-rust-in-our-high-performance-java-database/) with no dependencies and zero garbage collection.
+QuestDB is hardware efficient, with quick setup and operational efficiency.
 
-QuestDB supports schema-agnostic streaming ingestion using the InfluxDB line protocol and a REST API for bulk imports and exports.
-The QuestDB SQL Web Console is an interactive SQL editor facilitating CSV import. Finally, QuestDB also includes the Postgres Wire Protocol
-for programmatic queries.
+> Ready to go? Check out our
+> [quick start guide](https://questdb.io/docs/quick-start/).
 
-Popular tools that integrate with QuestDB include [Apache Kafka](https://questdb.io/docs/third-party-tools/kafka/questdb-kafka/), [Grafana](https://questdb.io/docs/third-party-tools/grafana/),
-[Superset](https://questdb.io/docs/third-party-tools/superset/), [Telegraf](https://questdb.io/docs/third-party-tools/telegraf/) and [Apache Flink](https://questdb.io/docs/third-party-tools/flink/).
+<p>&nbsp;</p>
 
 <div align="center">
-  <a href="https://demo.questdb.io">
-    <img alt="QuestDB Web Console showing a SQL statement and query result" src="https://raw.githubusercontent.com/questdb/questdb/master/.github/console.png" width="600" />
+  <a href="https://demo.questdb.io/">
+    <img alt="QuestDB Web Console showing a SQL statement and query result" src="https://raw.githubusercontent.com/questdb/questdb/master/.github/console.png" width="900" />
   </a>
+  <p><em>QuestDB Web Console</em></p>
 </div>
 
+<p>&nbsp;</p>
+
+## Benefits of QuestDB
+
+QuestDB excels with:
+
+- financial market data
+- IoT sensor
+- real-time dashboards
+- time-series data with high data cardinality
+
+Feature highlights include:
+
+- SQL with powerful time-series extensions: PGWire, HTTP API
+- High-speed ingestion through the InfluxDB Line Protocol over TCP or HTTP
+- Strong and efficient performance on limited hardware (Raspberry Pi, 4 workers)
+- Columnar storage format (native or
+  [Apache Parquet](https://questdb.io/glossary/apache-parquet/)), partitioned
+  and ordered by time
+- Responsive and intuitive Web Console for query and data management, with error
+  handling
+- Excellent performance with
+  [high data cardinality](https://questdb.io/glossary/high-cardinality/)
+
+And why use a time-series database?
+
+Beyond performance and efficiency, with a specialized [time-series database](https://questdb.io/glossary/time-series-database/), you don't need to worry about:
+
+- out-of-order data
+- duplicates
+- exactly one semantics
+- frequency of ingestion
```

Comment:
I would change "frequency of ingestion". Themes we can explore that are interesting:
streaming data (low latency)
dealing with a large number of concurrent requests
Minimal time between data ingestion and ability to read the data